_includes/v19.2/sql/function-special-forms.md

```diff
@@ -1,5 +1,6 @@
 | Special form                                              | Equivalent to                               |
 |-----------------------------------------------------------|---------------------------------------------|
+| `AT TIME ZONE`                                            | `timezone()`                                |
 | `CURRENT_CATALOG`                                         | `current_catalog()`                         |
```

Comment:
Sorry, comments got dropped!
Couldn't find this one on the SQL grammar page, so that may need an update.